mindmeld/cli.py

```diff
@@ -540,6 +543,30 @@ def _get_duckling_pid():
     return pid
 
 
+@shared_cli.command("annotate", context_settings=CONTEXT_SETTINGS)
+@click.option(
+    "--app-path", required=True, help="The application's path.",
+)
+def annotate(app_path):
+    """Runs the annotation command of the Auto Annotator."""
+    config = get_auto_annotator_config(app_path=app_path)
+    annotator = create_annotator(app_path=app_path, config=config)
+    annotator.annotate()
+    logger.info("Annotation Complete.")
+
+
+@shared_cli.command("unannotate", context_settings=CONTEXT_SETTINGS)
+@click.option(
+    "--app-path", required=True, help="Needed to locate the application config file.",
+)
+def unannotate(app_path):
+    """Runs the unannotation command of the Auto Annotator."""
+    config = get_auto_annotator_config(app_path=app_path)
+    annotator = create_annotator(app_path=app_path, config=config)
+    annotator.unannotate()
+    logger.info("Unannotation Complete.")
```

Comment:
I don't think 'unannotation' is a word, maybe we can say 'Annotation Removal Compete'?